sql-plan-management.md

```diff
@@ -248,14 +321,88 @@ However, TiDB does not automatically capture bindings for the following types of
 
+### Filter out bindings
+
+This function allows you to configure a blocklist to filter out queries that meet the requirements. A blocklist has three dimensions, table name, frequency, and user name.
+
+#### Usage
+
+Insert filtering conditions into the system table `mysql.capture_plan_baselines_blacklist`. Then the filtering conditions take effect in the entire cluster immediately.
+
+```sql
+-- Filter by table name
+ INSERT INTO mysql.capture_plan_baselines_blacklist(filter_type, filter_value) VALUES('table', 'test.t');
+
+-- Filter by database name and table name through wildcards
+ INSERT INTO mysql.capture_plan_baselines_blacklist(filter_type, filter_value) VALUES('table', 'test.table_*');
+ INSERT INTO mysql.capture_plan_baselines_blacklist(filter_type, filter_value) VALUES('table', 'db_*.table_*');
+
+-- Filter by frequency
+ INSERT INTO mysql.capture_plan_baselines_blacklist(filter_type, filter_value) VALUES('frequency', '2');
+
+-- Filter by user name
+ INSERT INTO mysql.capture_plan_baselines_blacklist(filter_type, filter_value) VALUES('user', 'user1');
+```
+
+| **Dimension** | **Dimension name** | **Description**                                                     | Remarks                                                     |
+| :----------- | :----------- | :----------------------------------------------------------- | ------------------------------------------------------------ |
+| Table name         | table        | Filter by table name. Each filtering rule is in the form of `db.table` and supports wildcards. For details about the rules, see [Plain table names](/table-filter.md# plain-table-names) and [Wildcards](/table-filter.md#wildcards). | Case insensitive. If the table name contains illegal content, the log returns a warning message `[sql-bind] failed to load mysql.capture_plan_baselines_blacklist`. |
+| Frequency         | frequency    | SQL statements executed more than once are captured by default. You can set a high frequency to capture statements more frequently. | Setting frequency to a value smaller than 1 is considered illegal, and the log returns a warning message `[sql-bind] frequency threshold is less than 1, ignore it`. If multiple frequency filter rules are inserted, the value with the highest frequency prevails. |
+| User name       | user         | Statements executed by blocklisted users are not captured.                           | If multiple users execute the same statement, this statement is not captured only if the user names are all in the blocklist. |
+
+> **Note:**
+>
+> - Modifying a blocklist requires the super privilege.
+>
+> - If the blocklist contains illegal filter content, TiDB returns warning message `[sql-bind] unknown capture filter type, ignore it` in the log.
+
+### Prevent rollback of execution plans during an upgrade
+
+When upgrading a TiDB cluster, you can use baseline capturing to prevent rollback of execution plans by performing the following steps:
```

Comment:
```suggestion
When upgrading a TiDB cluster, you can use baseline capturing to prevent regression of execution plans by performing the following steps:
```